### ISSUE TITLE
`helmet()` is not a function

### ISSUE THREAD

**Issue by JeffreyArt1** [helmetjs/helmet#348]
I'm currently using Nest.js 8.1.5.

I have tested helmet `4.6.0` and `5.0.1` with the same results.

This is my server file:

```Typescript
import { NestFactory } from '@nestjs/core'
import helmet from 'helmet';

async function bootstrap() {
  const app = await NestFactory.create(AppModule);

  app.enableCors();
  app.use(helmet());

  await app.listen(AppModule.applicationPort);
}

bootstrap()
```

When I try to run it, it prints this to console.
```
TypeError: (0 , helmet_1.default) is not a function
    at bootstrap (/Users/jeffreymesa/Desktop/dev.nosync/yp-micro/yp-lottery-api-gateway/src/main.ts:14:17)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

**Comment by EvanHahn** [comment]
Sorry you're running into this. What does your `tsconfig.json` look like?

**Comment by daniel-ackermann** [comment]
I have the same error. I use Nestjs 8.2.5 with this tsconfig.json:

```
{
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "target": "es2017",
    "sourceMap": true,
    "outDir": "./dist",
    "baseUrl": "./",
    "incremental": true,
    "skipLibCheck": true,
    "strictNullChecks": false,
    "noImplicitAny": false,
    "strictBindCallApply": false,
    "forceConsistentCasingInFileNames": false,
    "noFallthroughCasesInSwitch": false
  }
}
```

**Comment by EvanHahn** [comment]
In the short term, you can do one of the following:

1. Try `import helmet = require('helmet')`
1. Enable `esModuleInterop` in your `tsconfig`
1. Downgrade to Helmet v4

In the long term, I'll look into what's causing this. I suspect it will be fixed in an upcoming version (see #345), but I have been busy with other things and haven't had time to look yet. Apologies!

**Comment by daniel-ackermann** [comment]
Thank you for your fast solution! That works fine for me!

And sorry for the duplication of #344

**Comment by JeffreyArt1** [comment]
> Sorry you're running into this. What does your `tsconfig.json` look like?

mine looks like this:

```JSON
{
  "compilerOptions": {
    "module": "commonjs",
    "declaration": true,
    "removeComments": true,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "allowSyntheticDefaultImports": true,
    "target": "es2017",
    "sourceMap": true,
    "outDir": "./dist",
    "baseUrl": "./",
    "incremental": true,
    "skipLibCheck": true,
    "strictNullChecks": false,
    "noImplicitAny": false,
    "strictBindCallApply": false,
    "forceConsistentCasingInFileNames": false,
    "noFallthroughCasesInSwitch": false
  }
}
```

I recently added `allowSyntheticDefaultImports` to see if that could be a solution.

**Comment by JeffreyArt1** [comment]
> In the short term, you can do one of the following:
> 
> 1. Try `import helmet = require('helmet')`
> 2. Enable `esModuleInterop` in your `tsconfig`
> 3. Downgrade to Helmet v4
> 
> In the long term, I'll look into what's causing this. I suspect it will be fixed in an upcoming version (see #345), but I have been busy with other things and haven't had time to look yet. Apologies!

These solutions worked for me with helmet `4.6.0`:
- `import helmet = require('helmet')`
- `import * as helmet from 'helmet'`

Tried using `esModuleInterop` in my `tsconfig` (with version 5) with no luck :(

**Comment by EvanHahn** [comment]
This should be fixed in an upcoming version. Please stay tuned!

**Comment by EvanHahn** [comment]
I just released `helmet@5.0.2` which should fix this problem. Please try updating to see if that fixes things for you!

**Comment by JeffreyArt1** [comment]
Upgraded and tested, so far so good. Thanks!
